build.gradle

```diff
@@ -56,6 +57,24 @@ java {
     }
 }
 
+tasks.withType(AbstractArchiveTask).configureEach {
+    preserveFileTimestamps = false
+    reproducibleFileOrder = true
+    filePermissions {
+        unix('rw-r--r--')
+    }
+    dirPermissions {
+        unix('rwxr-xr-x')
+    }
+}
+
+
+jar {
+    manifest {
+        attributes("Bundle-Version": version)
```

Comment:
you might want to also add `Implementation-Version` (https://github.com/ben-manes/caffeine/issues/889)

Reply:
Done. Thanks.